#! /bin/sh /usr/share/dpatch/dpatch-run
## hp-check-cups-config.dpatch by  Mark Purcell <msp@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: https://bugs.launchpad.net/bugs/565689

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' hplip-3.10.5~/check.py hplip-3.10.5/check.py
--- hplip-3.10.5~/check.py	2010-05-13 09:45:16.000000000 +1000
+++ hplip-3.10.5/check.py	2010-05-19 21:46:01.000000000 +1000
@@ -339,9 +339,7 @@
         if status == 0:
             log.info("Version: %s" % output.strip())
         else:
-            log.error("Version: (Not available. CUPS may not be installed or not running.)")
-            cups_ok = False
-            num_errors += 1
+            log.warn("Version: (Not available - cups-config. Unable to determine installed version of CUPS.)")
 
     if cups_ok:
         cups_conf = '/etc/cups/cupsd.conf'
